odl/trafos/fourier.py

```diff
@@ -7,9 +7,11 @@
 # obtain one at https://mozilla.org/MPL/2.0/.
 
 """Discretized Fourier transform on L^p spaces."""
-
 from __future__ import print_function, division, absolute_import
+from collections.abc import Iterable
+
 import numpy as np
+from pynfft.nfft import NFFT
```

Comment:
This needs to be include guarded so that users can run ODL without it. Further, I would personally put the nfft in a different file for now.

Reply:
I put the nfft in a different file. I applied the mechanism you used for `pywt` and `pyfftw` to `pynfft`, even if the bindings file is way smaller. Tell me what you think about it.